pkg/common/common.go

```diff
@@ -41,4 +45,11 @@ const (
 	// references with any that already exist on the object rather than replace the owner references. Further
 	// the controller in the owner reference will not be set.
 	MultipleOwnersLabel = "operator.tigera.io/multipleOwners"
+	BPFVolumeName       = "bpffs"
 )
+
+func BPFDataplaneEnabled(installation *operatorv1.InstallationSpec) bool {
```

Comment:
I think this could very well be added as a function attached to the `Installation` struct instead of a helper here.

e.g., 

```
installation.BPFEnabled()
```

We don't often do this (attach functions to our API structures) but this one seems like a good candidate since it's very much just a shortcut to a field on that structure.

Reply:
Thanks @caseydavenport - nice idea - have implemented as suggested